src/main/webapp/app/tasks/task-list/sort-tasks.pipe.spec.ts

```diff
@@ -0,0 +1,61 @@
+import {DEFAULT_TASK_STATUS_ORDER, SortTasksPipe} from './sort-tasks.pipe';
+import {Task, TaskStatus} from "app/tasks/task";
+
+describe('SortTasksPipe', () => {
+  it('create an instance', () => {
+    const pipe = new SortTasksPipe();
+    expect(pipe).toBeTruthy();
+  });
+
+  it('should use default sorting order with done on the top if no other order provided', () => {
+    const pipe = new SortTasksPipe();
+    const tasks: Task[] = [
+      { id: '1', name: '1', status: TaskStatus.Blocked },
+      { id: '2', name: '2', status: TaskStatus.InProgress },
+      { id: '3', name: '3', status: TaskStatus.Done },
+      { id: '4', name: '4', status: TaskStatus.Todo },
+      { id: '5', name: '5', status: TaskStatus.Cancelled },
+    ];
+    const result = pipe.transform(tasks).map(({status}: Task) => status);
+    expect(result).toEqual(DEFAULT_TASK_STATUS_ORDER);
```

Comment:
Better check would be for the actual tasks ordered correctly. Strictly speaking it would be unnoticed if the pipe would change the id "1" task status to "todo"